Translated Readme.md

BigFaceMaster opened the pull request.

Commit:
Init

Commit:
Finished the Symmetric Difference

Commit:
修改人称词

Commit:
优化

Commit:
Finished Inventory Update

Commit:
test

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
Undone

Commit:
Format by pangu

Commit:
与其他页面的相同信息保持一致

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
No Repeats Please

Commit:
Modified by reviwer

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
modified the space of chinese && code

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
Modified by reviwer

Commit:
Delete the space

Commit:
Modified by reviwer

Commit:
Modified by reviwer

Commit:
Recover space

Commit:
Translated ReadMe

BigFaceMaster commented:
需将`08-coding-interview-prep/algorithms.json`所有合并之后才合并此pr

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

huluoyang commented on 08-coding-interview-prep/algorithms.json:
```diff
@@ -172,53 +173,53 @@
-        "Return the number of total permutations of the provided string that don't have repeated consecutive letters. Assume that all characters in the provided string are each unique.",
-        "For example, <code>aab</code> should return 2 because it has 6 total permutations (<code>aab</code>, <code>aab</code>, <code>aba</code>, <code>aba</code>, <code>baa</code>, <code>baa</code>), but only 2 of them (<code>aba</code> and <code>aba</code>) don't have the same letter (in this case <code>a</code>) repeating.",
-        "Remember to use <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code."
+        "把一个字符串中的字符重新排列生成新的字符串，返回新生成的字符串里没有连续重复字符的字符串个数。连续重复只以单个字符为准。",
```
语句不通，符号使用混乱。

huluoyang commented on 08-coding-interview-prep/algorithms.json:
```diff
@@ -172,53 +173,53 @@
-        "Remember to use <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code."
+        "把一个字符串中的字符重新排列生成新的字符串，返回新生成的字符串里没有连续重复字符的字符串个数。连续重复只以单个字符为准。",
+        "例如：<code>aab</code>应该返回 2， 因为它总共有 6 种排列方式： <code>aab</code>， <code>aab</code>， <code>aba</code>， <code>aba</code>， <code>baa</code>， <code>baa</code>，但是其中只有 2 个没有连续重复的字符（ 字符 a 是本例中的重复字符 ）：<code>aba</code>，<code>aba</code>",
+        "如果你遇到了困难，请点击 <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514'target='_blank'> 帮助 </a>。你可以找人” 结对编程 “，但不要因此放弃思考。"
```
符号使用混乱

huluoyang commented on 08-coding-interview-prep/algorithms.json:
```diff
@@ -99,36 +100,36 @@
-        "Compare and update the inventory stored in a 2D array against a second 2D array of a fresh delivery. Update the current existing inventory item quantities (in <code>arr1</code>). If an item cannot be found, add the new item and quantity into the inventory array. The returned inventory array should be in alphabetical order by item.",
-        "Remember to use <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code."
+        "创建一个<a href='https://baike.baidu.com/item/%E4%BA%8C%E7%BB%B4%E6%95%B0%E7%BB%84' target='_blank'>二维数组</a>，比较并更新存储在<code>二维数组</code>中的 ”库存“ 元素，然后并将其与新产生的第二个二维数组进行对比，更新当前的 ”库存“ 项的数量（<code>arr1</code>），如果找不到这个对比对象，那么将新的对象和数量添加到 “库存” 数组中。注意：返回的 “库存” 数组应该是按照数组元素的首字母顺序排序",
+        "如果你遇到了困难，请点击<a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>帮助</a>。你可以找人“结对编程“，但不要因此放弃思考。",
```
符号使用混乱

huluoyang reviewed (changes requested):
关于引号的使用较为混乱，请仔细对照下。

Commit:
Merge branch 'translate' of https://github.com/leejay1992/challenges into readme

Commit:
Merge branch 'translate' of github.com:FreeCodeCampChina/challenges into translate

Commit:
Merge branch 'translate' of https://github.com/leejay1992/challenges into readme

Commit:
Modified by reviwer

S1ngS1ng commented:
![image](https://user-images.githubusercontent.com/13592559/43853707-c572076e-9af5-11e8-925b-0eb76910c229.png)

不要 `pull` 了。如果不好处理，重开新的 branch 吧

huluoyang committed:
更新文件 README.md

huluoyang committed:
更新文件 README.md

rainpure committed:
functional-programming.json complete

rainpure committed:
Accept Merge Request #10: (functional-programming -> translate)
Merge Request: functional-programming.json complete
Created By: @黄雨纯
Accepted By: @黄雨纯
URL: https://freecodecamp.coding.net/p/challenges/git/merge/10

rainpure committed:
Revert "functional-programming.json complete"

S1ngS1ng committed:
Accept Merge Request #11: (revert-10-functional-programming -> translate)
Merge Request: Revert "functional-programming.json complete"
Created By: @胡落阳
Accepted By: @柳星
URL: https://freecodecamp.coding.net/p/challenges/git/merge/11

Commit:
Merge branch 'translate' of https://e.coding.net/freecodecamp/challenges into readme

Commit:
algorithms json link
